api/src/main/java/io/opencensus/metrics/MetricRegistry.java

```diff
@@ -159,6 +159,64 @@ public DerivedDoubleGauge addDerivedDoubleGauge(
   @ExperimentalApi
   public abstract DerivedDoubleGauge addDerivedDoubleGauge(String name, MetricOptions options);
 
+  /**
+   * Builds a new long cumulative to be added to the registry. This is more convenient form when you
+   * want to manually increase values as per your service requirements.
```

Comment:
"This is more convenient form" -> "This form is more convenient" or "This is a more convenient form" - here and below.

Reply:
SG, updated.